tests/functional/spec/placeholder_option_label_from_value.spec.js

```diff
@@ -0,0 +1,34 @@
+import MandatoryCheckboxPage from "../generated_pages/placeholder_option_label_from_value/mandatory-checkbox.page";
+import RecoveryQuestionCheckboxBlockPage from "../generated_pages/placeholder_option_label_from_value/recovery-question-checkbox-block.page";
+import MandatoryRadioPage from "../generated_pages/placeholder_option_label_from_value/mandatory-radio.page";
+import RecoveryQuestionRadioBlockPage from "../generated_pages/placeholder_option_label_from_value/recovery-question-radio-block.page";
+
+describe("Option label value check", () => {
+  beforeEach("Load the survey", () => {
+    browser.openQuestionnaire("test_placeholder_option_label_from_value.json");
+  });
+
+  it("Given multiple checkbox options are provided, when I select a single answer and go to the next page, then the question title contains the label text of the answer I selected", () => {
+    $(MandatoryCheckboxPage.head()).click();
+    $(MandatoryCheckboxPage.submit()).click();
+    expect($(RecoveryQuestionCheckboxBlockPage.questionText()).getText()).to.contain("Head");
+  });
+
+  it("Given multiple checkbox options are provided, when I select more than one answer and go to the next page, then first option in radio is piped from checkbox", () => {
+    $(MandatoryCheckboxPage.head()).click();
+    $(MandatoryCheckboxPage.body()).click();
+    $(MandatoryCheckboxPage.submit()).click();
+    const answerPipedOption = $(MandatoryRadioPage.answerBodyPartPipedLabel()).selector;
+    expect($(answerPipedOption).getText()).to.contain("Head (piped)");
```

Comment:
You don't need to get the selector specifically.
```suggestion
    expect($(MandatoryRadioPage.answerBodyPartPipedLabel()).getText()).to.contain("Head (piped)");
```